modules/travel_pay/app/services/travel_pay/appointments_service.rb

```diff
@@ -45,14 +45,17 @@ def find_by_date_time(date_string, appointments)
         Rails.logger.error(message: 'Invalid appointment time provided (appointment time cannot be nil).')
         raise ArgumentError, message: 'Invalid appointment time provided (appointment time cannot be nil).'
       elsif date_string.present?
-        parsed_date_time = DateTime.parse(date_string)
-
+        parsed_date_time = DateUtils.strip_timezone(date_string)
         appointments.find do |appt|
-          !appt['appointmentDateTime'].nil? &&
-            parsed_date_time == DateTime.parse(appt['appointmentDateTime'])
+          begin
+            parsed_appt_time = DateUtils.strip_timezone(appt['appointmentDateTime'])
```

Comment:
Is this one still valid?

Reply:
Oh wait, this is even a different casing than the pre-transform FE param, `appointmentDatetime` vs `appointmentDateTime`.

Reply:
yes, this one comes from the appointment, not the parameters